### ISSUE TITLE
Fix broken links in docs

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4257]
|<img alt="mulka's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| mulka reported|
|-|-|
|Trac ID|trac#4257|
|Type|defect|
|Created|2010-02-05 21:03:27Z|
|Branch|https://github.com/twisted/twisted/tree/broken-core-links-4257|

Run a scan for broken links in the docs, and fix them.

Here's an example: 

Near the bottom of this page:
http://twistedmatrix.com/documents/current/core/howto/tutorial/intro.html

There is a link "Twisted Application Configuration" which should link to the glossary, but the link needs to go one level up.

<details><summary>Searchable metadata</summary>

```
trac-id__4257 4257
type__defect defect
reporter__mulka mulka
priority__normal normal
milestone__ 
branch__branches_broken_core_links_4257 branches/broken-core-links-4257
branch_author__jesstess jesstess
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1265403807000000 1265403807000000
changetime__1267475483000000 1267475483000000
version__None None
owner__jesstess jesstess
cc__khorn cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn commented|
|-|-|

there's another broken link in /core/development/index:

the link points to "development for the pb modules", but the directory does not exist

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn commented|
|-|-|

more broken links reported in [#4274](https://github.com/twisted/twisted/issues/4274)

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess commented|
|-|-|

(In [28537]) Branching to 'broken-core-links-4257'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess removed owner|
|-|-|

There are several open broken link tickets right now, so I restricted the scope of this one to doc/core/howto/tutorial/intro.xhtml and everything under doc/core/development.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set owner to @jesstess|
|-|-|

thx, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_7"></a>|@jesstess set status to `closed`|
|-|-|

(In [28597]) Merge broken-core-links-4257

Author: jesstess
Reviewer: thijs
Fixes: [#4257](https://github.com/twisted/twisted/issues/4257)

Fix broken links in doc/core/howto/tutorial/intro.xhtml and
doc/core/development/.
